lib/magazine_luiza_rewards_v2/api/products.rb

```diff
@@ -3,22 +3,54 @@
 module MagazineLuizaRewardsV2
   module Api
     class Products < Base
+      OUT_OF_STOCK = 'out of stock'
```

Comment:
```suggestion
      OUT_OF_STOCK = 'out of stock'.freeze
```

Reply:
cop suggestion is without '.freeze'

Reply:
I didn't see the frozen literal string comment 😅 please disregard it